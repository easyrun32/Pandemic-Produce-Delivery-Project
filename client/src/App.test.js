import React from "react";
import { shallow } from "enzyme";
import toJson from "enzyme-to-json";
import { App } from "./App";
//commenting
it("renders correctly", () => {
  const MockFunction = () => {};

  const MockProps = {
    id: "2XGqN0k31F7T12SV2",
    privilege: false,
    email: "TEST@gmail.com",
    createdAt: { seconds: 1597934755, nanoseconds: 995000000 },
    displayName: "A GUY WHO'S PRACTICING TESTING LOL",
  };
  const wrapper = shallow(
    <App user={MockProps} setCurrentUser={MockFunction} />
  );
  expect(toJson(wrapper)).toMatchSnapshot();
});
