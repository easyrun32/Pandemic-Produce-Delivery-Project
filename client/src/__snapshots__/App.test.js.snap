// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div>
  <Connect(Header) />
  <Switch>
    <Route
      component={[Function]}
      exact={true}
      path="/"
    />
    <Route
      component={
        Object {
          "$$typeof": Symbol(react.memo),
          "WrappedComponent": [Function],
          "compare": null,
          "displayName": "Connect(ShopPage)",
          "type": [Function],
        }
      }
      path="/shop"
    />
    <Route
      component={[Function]}
      path="/about"
    />
    <Route
      exact={true}
      path="/signin"
      render={[Function]}
    />
    <Route
      exact={true}
      path="/admin/add"
      render={[Function]}
    />
    <Route
      exact={true}
      path="/admin/checkout"
      render={[Function]}
    />
    <Route
      component={
        Object {
          "$$typeof": Symbol(react.memo),
          "WrappedComponent": [Function],
          "compare": null,
          "displayName": "Connect(CheckoutPage)",
          "type": [Function],
        }
      }
      exact={true}
      path="/checkout"
    />
    <Route
      component={[Function]}
      exact={true}
      path="/covid-glimpse"
    />
  </Switch>
</div>
`;
